components/offres/suggestions/SuggestionCard.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+
+import { IconName } from 'components/ui/IconComponent'
+import { DataTag } from 'components/ui/Indicateurs/DataTag'
+import { TypeOffre } from 'interfaces/offre'
+
+interface SuggestionCardProps {
+  type: TypeOffre
+  titre: string
+  motsCles: string
+  labelLocalite: string
+}
+
+export default function SuggestionCard({
+  type,
+  titre,
+  motsCles,
+  labelLocalite,
+}: SuggestionCardProps) {
+  function getTypeLabel(): string {
+    switch (type) {
+      case TypeOffre.EMPLOI:
+        return 'Offre d’emploi'
+      case TypeOffre.SERVICE_CIVIQUE:
+      case TypeOffre.IMMERSION:
+      case TypeOffre.ALTERNANCE:
+        return ''
+    }
+  }
+
+  return (
+    <div className='rounded-small shadow-s p-6'>
```

Comment:
```suggestion
    <div className='rounded-large shadow-s p-6'>
```
d'après la maquette/lama